### PR TITLE
Release 0.29.4

### PR DESCRIPTION
**Pull request by gersonnoboa**

### CODE REVIEW

File: SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK-Swift.h [hunk 1]
```diff
@@ -779,6 +796,8 @@ SWIFT_CLASS("_TtC11SalemoveSDK11QueueTicket")
 @end
 
 
+
+
```
Comment (gugalo) [inline SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK-Swift.h:800]: This is auto-generated?

**Review by dukhovnyi — APPROVED** [review]
🚀

**Review by gugalo — APPROVED** [review]

File: SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK-Swift.h (hunk 1 above)
Comment (gersonnoboa) [reply SalemoveSDK.xcframework/ios-arm64/SalemoveSDK.framework/Headers/SalemoveSDK-Swift.h:800]: Yep

**Comment by gersonnoboa** [comment]
!merge

**Pull request merged**